content/docs/examples/multicluster/icp/index.md

```diff
@@ -5,10 +5,10 @@ weight: 70
 keywords: [kubernetes,multicluster]
 ---
 
-This example demonstrates how to setup networking between two
+This example demonstrates how to setup network connectivity between two
 [IBM Cloud Private](https://www.ibm.com/cloud/private) clusters
 and then compose them into a multicluster mesh using a
-[single control plane topology](/docs/concepts/multicluster-deployments/#single-control-plane-topology).
+[single control plane topology with VPN connectivity](/docs/concepts/multicluster-deployments/#single-control-plane-with-vpn-connectivity).
```

Comment:
using a [single control plane with VPN connectivity](/docs/concepts/multicluster-deployments/#single-control-plane-with-vpn-connectivity) topology.

Reply:
Fixed in https://github.com/istio/istio.io/pull/3294/commits/711a37d7905982f52a60bed0d09944883e00bbe0.